app/src/main/java/com/android/sample/mlUtils/exercisesCriterions/AngleCriterionComments.kt

```diff
@@ -0,0 +1,39 @@
+package com.android.sample.mlUtils.exercisesCriterions
+enum class AngleCriterionComments(val description: String) {
+  SUCCESS("Success"),
+  NOT_IMPLEMENTED("Not implemented yet"),
+  NONE(""),
+  SHOULDER_HIP_KNEE_NOT_FLAT("Your back should be more right"),
+  SHOULDER_HIP_KNEE_NOT_BENDED("Try bending more your body"),
+  SHOULDER_HIP_KNEE_NOT_RIGHT("Bend your body to have a 90° angle at your hips."),
+  LEFT_WRIST_ELBOW_SHOULDER_NOT_FLAT("Keep your right elbow totally open"),
+  RIGHT_WRIST_ELBOW_SHOULDER_NOT_FLAT("Keep your left elbow totally open"),
+  BOTH_WRIST_ELBOW_SHOULDER_NOT_FLAT("Keep your elbows stretched and totally open"),
+  RIGHT_WRIST_ELBOW_SHOULDER_NOT_RIGHT(
+      "Your right wrist, elbow and shoulder should make a right angle."),
```

Comment:
If it is not too difficult maybe it would be a good idea to use string resources.